.travis.yml

```diff
@@ -2,33 +2,30 @@ dist: xenial
 
 language: csharp
 mono: none
-dotnet: 2.2
+dotnet: 3.0
 
 git:
     depth: false
 
-addons:
-    sonarcloud:
-        organization: "steamre"
-
 solution: SteamKit2/SteamKit2.sln
 
+before_install:
+    - sudo apt-get update
+    - sudo apt-get install -y dotnet-runtime-2.1
```

Comment:
You changed to `dotnet: 3.0`, but install 2.1 here?

Reply:
.NET SDK 3.0 w/ 3.0 runtime.

We still need a .NET 2.x runtime to run our tests which target `netcoreapp2.0`, and we did need it to run SonarScanner which targets `netcoreapp2.1`.

By default, .NET apps don't roll forward on major versions to a different runtime.